DEVELOPMENT.md

```diff
@@ -144,6 +144,24 @@ release: the actual image only gets pushed when there is a tag that
 corresponds to the version embedded in the source code. The
 Jenkinsfile ensures that.
 
+Release Checklist
+-----------------
+
+* Create a new `release-x.y` branch.
+* Run `hack/update-clear-linux-base.sh`.
+* Run `hack/set-version.sh vx.y.z` and commit the modified files.
+* Push to `origin`.
+* [Create a draft
+  release](https://github.com/intel/pmem-csi/releases/new) for that
+  new branch, including a change log gathered from new commits.
+* Review change log.
```

Comment:
I think more correct is "Review the change log"

Reply:
Indeed.

Reply:
Fixed. Please have another look and check also my replies.